docs/api/curl/UmbrellaConectorDataExchange.md

```diff
@@ -0,0 +1,223 @@
+## Data Exchange Tutorial
+In order to start exchanging data, it will be necessary to install the subset with the necessary components for data exchange:
+```
+helm install umbrella -f ./charts/umbrella/values-adopter-data-exchange.yaml ./charts/umbrella/ --namespace umbrella
+```
+
+#### Prerequisites
+As a Backend Service, we are using [Webhook.site](https://webhook.site/), as it is easier to work with. In the variables {{webhook}}, use your own one.
+> But you can also use the one given in the MXD [Backend-service](https://github.com/eclipse-tractusx/tutorial-resources/tree/main/mxd/backend-service)
```

Comment:
I don't quite understand why this Backend Service is necessary, could you please explain?
Isn't the install of the helm chart the "Backend Service"?

Reply:
I don't understand what you mean by the installation being the "backend service", which URL would be used then in the parameters that is asking for the Backend-Service? I wasn't able to figure that out. 
I would be delighted if you could help me with this. :smile:

Reply:
Sorry for the confusion, I might have only now understood your question from yesterday :)
I'm not firm on this, does this URL need to coincide with https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/values.yaml#L1083?
If this simple data backend should be reachable from outside the K8s cluster (for postman and curl requests), we could enable the ingress here https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/values.yaml#L1179.